en/deploy-tidb-cluster-across-multiple-kubernetes.md

```diff
@@ -0,0 +1,662 @@
+---
+title: Deploy a TiDB Cluster across Multiple Kubernetes Clusters
+summary: Learn how to deploy a TiDB cluster across multiple Kubernetes clusters.
+---
+
+# Deploy a TiDB Cluster across Multiple Kubernetes Clusters
+
+To deploy a TiDB cluster across multiple Kubernetes clusters refers to deploying **one** TiDB cluster on multiple network-interconnected Kubernetes clusters. Each component of the cluster is distributed on multiple Kubernetes clusters to achieve disaster recovery among Kubernetes clusters. The interconnected network of Kubernetes clusters means that Pod IP can be accessed in any cluster and between clusters, and Pod FQDN records can be parsed in any cluster and between clusters.
+
+## Prerequisites
+
+You need to configure the Kubernetes network and DNS so that the Kubernetes cluster meets the following conditions:
+
+- The TiDB components on each Kubernetes cluster can access the Pod IP of all TiDB components in and between clusters.
+- The TiDB components on each Kubernetes cluster can parse the Pod FQDN of all TiDB components in and between clusters.
+
+## Supported scenarios
+
+Currently supported scenarios:
+
+- Newly deployed a TiDB cluster across multiple Kubernetes clusters.
+- Deploy new clusters that enable this feature on other Kubernetes clusters and join the clusters that also enable this feature.
```

Comment:
```suggestion
- Deploy a new TiDB cluster that enables this feature on other Kubernetes clusters and join the cluster with another TiDB cluster that also enables this feature.
```

Reply:
@handlerww Please check whether my suggestion is correct. Thanks!

Reply:
Actually, we can deploy many clusters to join the initial cluster.  Should we cover this knowledge?

Reply:
Sure. How about this?

```suggestion
- Deploy new TiDB clusters that enable this feature on other Kubernetes clusters and join the initial TiDB cluster.
```